gunicorn/http/message.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, cfg, unreader, peer_addr):
         self.cfg = cfg
         self.unreader = unreader
         self.peer_addr = peer_addr
-        self.remote_addr = peer_addr[0]
+        # self.remote_addr = peer_addr[0]
```

Comment:
Without this the tests don't work at all, see https://github.com/benoitc/gunicorn/pull/2896 -- should be remove before merge.